sorc/build_all.sh

```diff
@@ -196,12 +201,14 @@ while [[ ${builds_started} -lt ${#build_jobs[@]} ]]; do
       if [[ -n "${build_jobs[${build}]+0}" && -z "${build_ids[${build}]+0}" ]]; then
          # Do we have enough processors to run it?
          if [[ ${_build_job_max} -ge $(( build_jobs[build] + procs_in_use )) ]]; then
-            if [[ "${build}" != "upp" ]]; then
-               "./build_${build}.sh" -j "${build_jobs[${build}]}" "${build_opts[${build}]:-}" > \
-                  "${logs_dir}/build_${build}.log" 2>&1 &
-            else
+            if [[ "${build}" == "upp" ]]; then
                "./build_${build}.sh" "${build_opts[${build}]}" > \
                   "${logs_dir}/build_${build}.log" 2>&1 &
+            else  # not upp
+               # double-quoting build_opts here will not work since it is a string of options
+               #shellcheck disable=SC2086
+               "./build_${build}.sh" ${build_opts[${build}]:-} -j "${build_jobs[${build}]}" > \
+                  "${logs_dir}/build_${build}.log" 2>&1 &
             fi
```

Comment:
The `build_upp.sh` script now accepts `-j N` flag, so this if/else block can be removed.

```suggestion
            # double-quoting build_opts here will not work since it is a string of options
            #shellcheck disable=SC2086
            "./build_${build}.sh" ${build_opts[${build}]:-} -j "${build_jobs[${build}]}" > \
               "${logs_dir}/build_${build}.log" 2>&1 &
```

Reply:
Thanks for catching this @DavidHuber-NOAA